src/main/webapp/app/shared/util/firebase/firebase-review-utils.tsx

```diff
@@ -602,21 +617,35 @@ export const buildCancerTypeNameReview = (
   return metaReview;
 };
 
+const getAssociationVariantStringFromUuids = (uuids: string[], mutations: Mutation[]) => {
+  return uuids
+    .map(uuid => {
+      if (uuid === PATHOGENIC_VARIANTS) {
+        return PATHOGENIC_VARIANTS;
+      }
```

Comment:
Could you add a comment here that we need to remove when working on https://github.com/oncokb/oncokb-pipeline/issues/389